WORKSPACE

```diff
@@ -1,9 +1,7 @@
 workspace(name = "com_github_google_benchmark")
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-
-git_repository(
-    name = "com_google_googletest",
-    commit = "3f0cf6b62ad1eb50d8736538363d3580dd640c3e",  # HEAD
-    remote = "https://github.com/google/googletest",
+http_archive(
+     name = "com_google_googletest",
+     urls = ["https://github.com/google/googletest/archive/master.zip"],
```

Comment:
should this be a specific commit so we don't end up with version mismatch issues as googletest releases?

Reply:
I figured that diamond dependency isn't an issue here because the benchmark library doesn't actually depend on gtest, only the benchmark tests do. So whatever version of gtest is brought in here does not get forwarded to a project that uses benchmark. Let me know if I'm confused on that and I'll look into adding a specific commit back.

Reply:
I'm more worried about tests starting to fail because of some change in the head version of googletest that is later reverted. It's unlikely, but i'd rather depend on a specific version.

something like https://github.com/google/googletest/archive/3f0cf6b62ad1eb50d8736538363d3580dd640c3e.zip in the URL should work.

Reply:
Done